docs/flow/tutorials/chaining.md

```diff
@@ -0,0 +1,74 @@
+---
+aliases:
+- /docs/agent/latest/flow/tutorials/chaining
+title: Chaining components
```

Comment:
I'd still like this to have a Prometheus-related title.

Reply:
Changed name to "Chaining Prometheus components".